ui/src/features/project/project-details/nodes/stage-node.tsx

```diff
@@ -46,29 +49,29 @@ export const StageNode = ({
         }}
       >
         <h3 className='flex items-center text-white justify-between'>
-          <div className='text-ellipsis whitespace-nowrap overflow-hidden'>
-            {stage.metadata?.name}
+          <div className='truncate pb-1'>{stage.metadata?.name}</div>
```

Comment:
`truncate` is identical to `text-ellipsis whitespace-nowrap overflow-hidden`.

`pb-1` is added to prevent descender letters (e.g. `g`, `j`, `p`) from being cut off at the bottom